### ISSUE TITLE
improve local usage to help with troubleshooting

### ISSUE THREAD

**Issue by teto** [nvim-neorocks/luarocks-tag-release#19]
to test the action locally I had to export all these variables (some referenced https://docs.github.com/en/actions/learn-github-actions/variables) + pass a dozen arguments:
```
export GITHUB_REPOSITORY=nvim-treesitter/nvim-treesitter
export GITHUB_SERVER_URL=https://github.com
export LUAROCKS_API_KEY=your-key
export GITHUB_REF_NAME=fake-ref-name
export GITHUB_SHA=fakesha_toto
# launch script 
result/bin/luarocks-tag-release nvim-treesitter 0.1 "" "neovim" "" "short desc" "detailed desc" "make" "./rockspec.template" 
```
we could probably make this easier

**Comment by mrcjkb** [comment]
I think this can be closed with #113 and #114 (see also #79).

@teto what do you think? Or does it need more refactoring for local integration testing?

**Comment by teto** [comment]
sure
